config/functions

```diff
@@ -318,6 +356,36 @@ listremoveitem() {
   echo "${tmp_array[@]}"
 }
 
+# check if a flag is enabled
+# $1: flag-name, $2: default (yes/no), $3: ingenious check (none,only-disable,only-enable)
+# set variable [flag]_ENABLED=(yes/no)
+# return 0 is flag is enabled, otherwise 2
```

Comment:
Comment is out-of-date, should be: `# set variable PKG_[flag-name]_ENABLED=(yes/no)`
Typos, should be: `# return 0 if flag is enabled, otherwise 1`